### PR TITLE
Setup limit orders page

### PR DESCRIPTION
**Pull request by shoom3301**
# Summary

Just skeleton for Limit orders page

### CODE REVIEW

**Review by anxolin — APPROVED** [review]
Again, approved to have it consolidated. Gave some feedback that might be nice to discuss and address in follow ups

File: src/cow-react/swap/dumb/SwapForm/index.tsx [hunk 1]
```diff
@@ -1,10 +1,10 @@
 import React from 'react'
 import { SwapFormProps } from 'cow-react/pages/NewSwap/typings'
 import * as styledEl from 'cow-react/pages/NewSwap/styled'
-import { CurrencyInputPanel } from 'cow-react/swap/dumb/CurrencyInputPanel'
-import { CurrencyArrowSeparator } from 'cow-react/swap/dumb/CurrencyArrowSeparator'
+import { CurrencyInputPanel } from 'cow-react/common/dumb/CurrencyInputPanel'
+import { CurrencyArrowSeparator } from 'cow-react/common/dumb/CurrencyArrowSeparator'
 import { swapPagePropsChecker } from 'cow-react/pages/NewSwap/propsChecker'
-import { AddRecipient } from 'cow-react/swap/dumb/AddRecipient'
+import { AddRecipient } from 'cow-react/common/dumb/AddRecipient'
```
[the remark this answers is not in the document]
Comment (shoom3301) [reply src/cow-react/swap/dumb/SwapForm/index.tsx:7]: Fixed

File: src/cow-react/pages/LimitOrders/styled.tsx [hunk 2]
```diff
@@ -0,0 +1,65 @@
+import styled from 'styled-components/macro'
+import { MEDIA_WIDTHS } from 'theme'
+import { Settings } from 'react-feather'
+
+export const Container = styled.div`
+  max-width: 460px;
+  width: 100%;
+`
+
+export const ContainerBox = styled.div`
```
[the remark this answers is not in the document]
Comment (shoom3301) [reply src/cow-react/pages/LimitOrders/styled.tsx:10]: 100% agree with you

File: src/custom/pages/App/index.tsx [hunk 3]
```diff
@@ -26,6 +26,7 @@ const SENTRY_TRACES_SAMPLE_RATE = process.env.REACT_APP_SENTRY_TRACES_SAMPLE_RAT
 
 // Async routes
 const PrivacyPolicy = lazy(() => import(/* webpackChunkName: "privacy_policy" */ 'pages/PrivacyPolicy'))
+const LimitOrders = lazy(() => import(/* webpackChunkName: "privacy_policy" */ 'cow-react/pages/LimitOrders'))
```
Comment (alfetopito) [inline src/custom/pages/App/index.tsx:29]: Forgot to update the webpackChunkName?

Comment (shoom3301) [reply src/custom/pages/App/index.tsx:29]: Thanks!

File: src/cow-react/common/dumb/CurrencyInputPanel/index.tsx [hunk 4]
```diff
@@ -1,18 +1,18 @@
 import React, { useCallback, useState } from 'react'
 import * as styledEl from './styled'
-import { CurrencySelectButton } from '../CurrencySelectButton'
+import { CurrencySelectButton } from '../../../swap/dumb/CurrencySelectButton'
```
[the remark this answers is not in the document]
Comment (shoom3301) [reply src/cow-react/common/dumb/CurrencyInputPanel/index.tsx:3]: Fixed

**Comment by elena-zh** [comment]
Just will add the note that it would be great to remove a blurred background on the page
image

**Pull request merged**